.travis.yml

```diff
@@ -1,20 +1,11 @@
 language: python
```

Comment:
why do you change this file?

Reply:
Because as soon as I commited [this](https://github.com/starkit/wsynphot/pull/6/commits/73dd4df76330601e9a1a21ac0bf4849a835c3da7), travis CI showed build is failing

Reply:
okay - I didn't realize that travis is already running on this env.

Reply:
And after interacting with @97amarnathk who has resolved the same issue in starkit, I came to know that:
>From what I have learnt recently, the .travis.yml files used by starkit are just generic ones, copied from astropy. These have’nt yet been customised for starkit.

So I customized it in similar manner of `starkit/.travis.yml`